### PR TITLE
change genre recommender's update method to sort only once

### PR DESCRIPTION
**Pull request by IdanTo1**
Signed-off-by: Idan Tomer <idan.tomer2@gmail.com>

### CODE REVIEW

File: include/Action.h [hunk 1]
```diff
@@ -11,9 +11,9 @@ enum ActionStatus{
 	PENDING, COMPLETED, ERROR
 };
 
-const std::string USER_EXISTS_ERR = "User already exists";
-const std::string INVALID_ALG_ERR = "Invalid algorithm was given";
-const std::string USER_DOESNT_EXISTS_ERR = "Requested user does not exist";
+const std::string USER_EXISTS_ERR = "the new user name is already taken";
+const std::string INVALID_ALG_ERR = "recommendation algorithm is invalid";
+const std::string USER_DOESNT_EXISTS_ERR = "username does not exist";
```
Comment (Hroy02) [inline include/Action.h:16]: Capital letters and recommendation in line 15

**Review by Hroy02 — CHANGES_REQUESTED** [review]
Grammar

File: include/Action.h (hunk 1 above)
Comment (IdanTo1) [reply include/Action.h:16]: Dude
It's copy paste 😂

**Review by Hroy02 — APPROVED** [review]
Perfect

**Pull request merged**